python/ray/projects/examples/README.md

```diff
@@ -0,0 +1,41 @@
+Ray Projects
+============
+
+To run these example projects, we first have to make sure the full
+repository is checked out into the project directory.
+
+Open Tacotron
+-------------
+
+```shell
+cd open-tacotron
+# Check out the original repository
+git init
+git remote add origin https://github.com/keithito/tacotron.git
+git fetch
+git checkout -t origin/master
```

Comment:
Don't you need to create a project here?

Reply:
It's already in the directory (the .rayproject in examples/<project-name>). The reason why I'm doing this complicated github stuff instead of just cloning it is because I want to clone the repo into these non-empty directories.

Reply:
Ahh I see. Looks good then!

Reply:
Ok great, it can be merged when the tests passed :)